### PR TITLE
🔀 💄 Merge Login branch containing updated Login UI

### PR DESCRIPTION
**Pull request by caaaramia**
Implemented new Login UI based on final app prototype, and split created widgets into separate classes for shared use in other features.

### CODE REVIEW

**Comment by Alexander-Muendesi** [comment]
I think just comment out that authentication test which is failing, put a mock one in place and I will deal with it myself. Sure something changed which needs to be fixed

File: libs/redux_comp/pubspec.yaml [hunk 1]
```diff
@@ -13,10 +13,11 @@ dependencies:
...
   # amplify
-  amplify_flutter: ^0.4.0
-  amplify_auth_cognito: ^0.4.0
-  amplify_datastore: ^0.4.0
-  amplify_api: ^0.4.0
+  amplify_flutter: ^0.5.1
+  amplify_auth_cognito: ^0.5.1
+  amplify_datastore: ^0.5.1
+  amplify_api: ^0.5.1
+  amazon_cognito_identity_dart_2: ^2.1.1
```
Comment (MichaelDuPlessis) [inline libs/redux_comp/pubspec.yaml:20]: Try removing this l

**Comment by MichaelDuPlessis** [comment]
> I think just comment out that authentication test which is failing, put a mock one in place and I will deal with it myself. Sure something changed which needs to be fixed

The problem doesn't seem to be the test it seems to be some dependency isseus

**Comment by Alexander-Muendesi** [comment]
> > I think just comment out that authentication test which is failing, put a mock one in place and I will deal with it myself. Sure something changed which needs to be fixed
> 
> The problem doesn't seem to be the test it seems to be some dependency isseus

I think even though you fixed the dependency stuff that test was still a problem

**Comment by MichaelDuPlessis** [comment]
@Alexander-Muendesi I see that flutter isn't happy with how Amplify generates its files, is there any way that we can disable flutter analyse from looking at the amplify folder.

**Comment by Alexander-Muendesi** [comment]
let me have a look, I am not entirely sure

**Comment by Alexander-Muendesi** [comment]
@MichaelDuPlessis  where those models there before or they are being introduced with this PR?

**Review by arlo216 — APPROVED** [review]

**Pull request merged**